frontend/views/Home/Home.tsx

```diff
@@ -136,10 +169,33 @@ function Home(props: Props): React.ReactElement {
           )}
         </Tab.Screen>
       </Tab.Navigator>
-      <FAB icon="plus" style={fabStyles as any} onPress={onAdd} />
+      <InviteFAB onPress={() => onAdd()} />
       <Portal>
-        <Modal visible={inviteModal} onDismiss={() => setInviteModal(false)}>
-          <Text>Example Modal. Click outside this area to dismiss.</Text>
+        <Modal
+          visible={inviteModal}
+          contentContainerStyle={
+            theme === "Dark" ? styles.modalDark : styles.modalLight
+          }
```

Comment:
Isn't this already inherited?

Reply:
If I comment it out it removes the background so I am going to say no. It is the style for the content, not the modal itself.